roles/rook/templates/03_CephBlockPool.yml.j2

```diff
@@ -0,0 +1,20 @@
+{% for pool_name in rook_cephblockpools %}
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: {{ pool_name }}
+  namespace: {{ rook_namespace }}
+# https://rook.io/docs/rook/latest-release/CRDs/Block-Storage/ceph-block-pool-crd/#spec
+spec:
+  failureDomain: host
```

Comment:
consider to parametrize this option as well

Reply:
:+1: